src/create-with-theme.js

```diff
@@ -1,31 +1,26 @@
 const React = require('react');
-const PropTypes = require('prop-types');
 const channel = require('./channel');
+const createThemeListener = require('./create-theme-listener');
 
 function createWithTheme(CHANNEL = channel) {
+  const themeListener = createThemeListener(CHANNEL);
   return Component =>
     class withTheme extends React.Component {
```

Comment:
then, it will be `withTheme(Comp)` because its function, and `WithTheme(Comp)` in react dev tools, and its confusing. lets stick to `withTheme` because its even more against common practice to have a normal function (not a class) with capital letter